src/modules/CRUD/components/Details/PropertyItem/Type/ArrayType/ArrayType.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getObjectOrderedValue, isTableObject } from '../../../../../services/utils';
+
+import Type from '..';
+import NoValue from '../../NoValue';
+
+const ArrayType = props => {
+  const {
+    display_value: displayValue,
+    schema,
+    schema: {
+      items: {
+        properties,
+        type,
+      },
+    },
+    t,
+    ui_schema: uiSchema,
+    ui_schema: {
+      items: {
+        'ui:order': UIOrder,
+      } = {},
+    } = {},
+  } = props;
+
+  if (displayValue === null || !displayValue.length) {
+    return <NoValue t={t} />;
+  }
+
+  if (type === 'object') {
+    const orderedValue = displayValue.map(value => getObjectOrderedValue(value, UIOrder));
+    if (isTableObject(orderedValue)) {
+      const columns = Object.keys(orderedValue[0]).map(value => properties[value].title);
+      return (
+        <table className="details__table">
+          <thead>
+            <tr>
+              {columns.map((th, i) => (
+                // eslint-disable-next-line react/no-array-index-key
+                <th key={`${th}${i}`}>{th}</th>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {orderedValue.map((row, i) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <tr key={i}>
+                {Object.keys(row).map((item, j) => (
+                  // eslint-disable-next-line react/no-array-index-key
+                  <td key={`${item}${j}`}>
+                    <Type
+                      display_value={row[item]}
+                      ui_schema={uiSchema.items[item]}
+                      schema={schema.items.properties[item]}
+                    />
+                  </td>
+                ))}
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      );
+    }
```

Comment:
This could easily be another sub component.

Reply:
Done